meta/main.yml

```diff
@@ -126,21 +126,10 @@ galaxy_info:
   #  - 9.3
   - name: Ubuntu
     versions:
-  #  - all
-  #  - lucid
-  #  - maverick
-  #  - natty
-  #  - oneiric
-  #  - precise
-  #  - quantal
-  #  - raring
-  #  - saucy
     - trusty
-  #  - utopic
-  #  - vivid
-  #  - wily
     - xenial
-  #  - yakkety
+    - bionic
```

Comment:
For me this can be also a separate patch to show new compatibility, that can be done after the 'main' molecule patch.